products/policy-controller.md

```diff
@@ -0,0 +1,81 @@
+---
+title: policy-controller
+category: server-app
+tags: go kubernetes
+permalink: /policy-controller
+releasePolicyLink: https://github.com/sigstore/policy-controller#support-policy
+releaseDateColumn: true
+eolColumn: false
```

Comment:
```suggestion
```

Not allowed as EOL dates are mandatory. And I can see that all releases have an EOL date so looks wrong.